lib/node/fullnode.js

```diff
@@ -227,6 +254,12 @@ class FullNode extends Node {
     await this.miner.open();
     await this.pool.open();
 
+    if (this.txindex)
```

Comment:
These (and the subsequent) if checks are going to have to be duplicated for any new indexers we or other implementers want to add huh? Probably doesn't need to be for this PR but would be nice if there was a way to do operations for each indexer. Maybe if indexers was an array that's passed to the node?